spec/in_app_receipt_spec.rb

```diff
@@ -12,11 +12,12 @@
         "purchase_date" => "2014-05-28 14:47:53 Etc/GMT",
         "original_transaction_id" => "140xxx867509",
         "original_purchase_date" => "2014-05-28 14:47:53 Etc/GMT",
-        "is_trial_period" => false,
+        "is_trial_period" => 'false',
```

Comment:
Since you've added support for `is_trial_period` in `in_app_receipt.rb`, please add a test for it in the `should output a hash with attributes` test below. Thanks!